### PR TITLE
pass devicePixelRatio to renderer

### LINKED ISSUE #712: use actual rendering resolution instead of CSS resolution
On mobile devices the images is aliased because it is not rendered in the native resolution.

### PR DESCRIPTION
**Pull request by stuikomma**
fixes #712

### CODE REVIEW

**Comment by stuikomma** [comment]
dpr changes when moving the browser window between monitors

**Comment by stuikomma** [comment]
Maybe update DPR only every x frames?

**Comment by stuikomma** [comment]
@adius you have different DPRs on your screens so you can test this better than the rest of us.

File: src/client/rendering/Renderer.coffee [hunk 1]
```diff
@@ -294,9 +296,15 @@ class Renderer
 		else
 			@threeRenderer.hasStencilBuffer = true
 
-		@threeRenderer.setSize @size().width, @size().height
 		@threeRenderer.autoClear = false
 
+	_updateSize: (forceUpdate) ->
+		devicePixelRatio = window.devicePixelRatio || 1
+		if forceUpdate or devicePixelRatio isnt @devicePixelRatio
```
Comment (adius) [inline src/client/rendering/Renderer.coffee]: Please use parentheses to clarify precedence

**Pull request merged**